busyApp/src/devBusyAsyn.c

```diff
@@ -35,38 +37,79 @@
 #include <recSup.h>
 #include <devSup.h>
 
+#include <epicsExport.h>
```

Comment:
Thanks for catching that.  This was the result of a cut and paste error, because in devAsynInt32.c epicsExport.h needs to be before the asyn include files, since they are in the same DLL.  But in busy epicsExport.h needs to be after the asyn include files.  I also moved busyRecord.h after epicsExport.h for consistency.